src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,14 +36,18 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        String name = authentication.getName();
+        Optional<User> optionalUser = userService.findByEmail(name);
+        ShoppingCart cart = shoppingCartService.getByUser(optionalUser.get());
```

Comment:
```suggestion
        ShoppingCart cart = shoppingCartService.getByUser(optionalUser.get());
```
let's throw an exception in case user is not found, the same for other similar cases